RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreationAlgo.cc

```diff
@@ -265,7 +265,7 @@ void CaloTowersCreationAlgo::setGeometry(const CaloTowerTopology* cttopo, const
     std::vector<int> tower28depths;
     int ndepths, startdepth;
     theHcalTopology->getDepthSegmentation(theHcalTopology->lastHERing()-1,tower28depths);
-    theHcalTopology->depthBinInformation(HcalEndcap,theHcalTopology->lastHERing()-1,ndepths,startdepth);
+    theHcalTopology->depthBinInformation(HcalEndcap,theHcalTopology->lastHERing()-1,1,1,ndepths,startdepth);
```

Comment:
it's fine for a work in progress.

Still, where is the  "plan1" wedge in units of iphi and zside?